includes/class-pb-book.php

```diff
@@ -185,6 +185,16 @@ static function getBookStructure( $id = '' ) {
 
 		$q = new \WP_Query();
 
+		/**
+		 * Fetch all pb_export meta values for this book
+		 */
+		global $wpdb;
+		$post_ids_to_export = array();
+		$sql = $wpdb->prepare( "SELECT post_id, meta_value FROM {$wpdb->postmeta} WHERE meta_key = %s", 'pb_export' );
+		foreach ( $wpdb->get_results( $sql, ARRAY_A ) as $val ) {
```

Comment:
OH!  I was trying to figure out what that error was trying to tell me.  I thought it was getting angry at the string interpolation used around `$wpdb->postmeta`.  It appears to be happy now on local.  I'll push up the change.